WireAPI/Sources/WireAPI/Models/UpdateEvent/ConversationEvent/ConversationAccessUpdateEvent.swift

```diff
@@ -35,11 +35,22 @@ public struct ConversationAccessUpdateEvent: Equatable, Codable {
     public let accessModes: Set<ConversationAccessMode>
 
     /// The new access roles.
-
-    public let accessRoles: Set<ConversationAccessRole>
+    public let accessRoles: Set<ConversationAccessRole>?
 
     /// The new legacy access role.
 
     public let legacyAccessRole: ConversationAccessRoleLegacy?
 
+    public init(conversationID: ConversationID,
+                senderID: UserID,
+                accessModes: Set<ConversationAccessMode>,
+                accessRoles: Set<ConversationAccessRole>?,
+                legacyAccessRole: ConversationAccessRoleLegacy?) {
```

Comment:
To remain consistent we should move the parameters on new lines.

Reply:
done in https://github.com/wireapp/wire-ios/pull/2028/commits/ead3b74b6aa5d7980d97061a124c80f8ac346e25